src/test/scala/com/fulcrumgenomics/umi/FilterConsensusReadsTest.scala

```diff
@@ -343,19 +327,25 @@ class FilterConsensusReadsTest extends UnitSpec {
       r1(ConsensusTags.PerRead.AbRawReadErrorRate) = abErr
       r1(ConsensusTags.PerRead.BaRawReadErrorRate) = baErr
       r1(ConsensusTags.PerBase.AbRawReadCount) = abDps
-      r1(ConsensusTags.PerBase.BaRawReadCount) = baDps
+      if (baDps.nonEmpty) r1(ConsensusTags.PerBase.BaRawReadCount) = baDps
```

Comment:
Could these conditions all be just `if baDp > 0`?  That way it relies less on getting empty arrays/strings.